ui/src/pages/home/apps-grid/apps-grid.tsx

```diff
@@ -27,18 +28,33 @@ export const AppsGrid = ({
   const {
     isLoading,
     error,
-    data: serverData,
-  } = useQuery<UserState, { message: string }>({
-    queryKey: ['app-state'],
-    queryFn: () =>
-      axios
-        .get(`/server/`)
-        .then((response) => {
-          return response.data;
-        })
-        .then((data) => {
-          return data;
-        }),
+    data: { serverData } = {
+      serverData: undefined,
+    },
+  } = useQuery<{ serverData: UserState; userData: User }, { message: string }>({
+    queryKey: ['app-state', 'user'],
+    queryFn: async () => {
+      // Gets server data
+      const getServerData = async () => {
+        const response = await axios.get(`/server/`);
+        return response.data;
+      };
+
+      // Gets user data
+      const getUserData = async () => {
+        const response = await axios.get(`/user/`);
+        return response.data;
+      };
```

Comment:
This can be removed with my recent update. userdata is now stored in global state